app.js

```diff
@@ -15,6 +15,10 @@ const users = require('./routes/users');
 const responses = require('./routes/responses');
 const { requiresStaffRole } = require('./middlewares/auth');
 
+// TESTING JERKINS
+var fnksn = (a,b,c) => {}
+var JSON = 'json'
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this override of "JSON". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.extensionengine.com/coding_rules#rule_key=javascript%3AS2424)